README.md

```diff
@@ -118,6 +118,42 @@ following command:
 $ npm -s run extract
 ```
 
+## Hash-based URLs for contexts
+
+This is an experimental feature. We have provided redirects using identifiers
+created from the SHA-256 hashed canonicalized JSON (JCS) of the JSON-LD contexts
```

Comment:
Yeah, maybe an `ni` multihash identifier (`mh`?) could allow the expression of that canonicalization step (known as "canon-hash" in multicodec space IIRC?).